articles/chart2d/chartModifierAPIs/zoomAndPan/ZoomPanModifier.md

```diff
@@ -19,6 +19,10 @@ There are several modes defined by the <xref:com.scichart.charting.ClipMode> enu
 - <xref:com.scichart.charting.ClipMode.ClipAtMax> - Forces the panning operation to stop suddenly at the maximum of the data, but expand at the minimum.
 - <xref:com.scichart.charting.ClipMode.ClipAtExtents> - Forces the panning operation to stop suddenly at the extents of the data.
 
+There are two touch events defined by the <xref:com.scichart.charting.modifiers.behaviors.GestureState> enumeration, which can be used to get or set current interaction state of the surface:
```

Comment:
This page needs the note about isDisallowInterceptTouchEvent as well.  Worth linking to the github page showing the code for the example that uses it.